ci-operator/step-registry/openshift-qe/installer/azure/openshift-qe-installer-azure-workflow.metadata.json

```diff
@@ -0,0 +1,29 @@
+{
```

Comment:
@shahsahil264 can you add me to the owners list to be able to review/merge this PR please?